backend/backend.Tests/AimTrainerGame/Controllers/AimTrainerGameControllerTest.cs

```diff
@@ -0,0 +1,46 @@
+using backend.AimTrainerGame.Controllers;
+using backend.AimTrainerGame.Models;
+using backend.Utils;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace backend.Tests.AimTrainerGame.Controllers;
+
+[TestSubject(typeof(AimTrainerGameController))]
+public class AimTrainerGameControllerTest
+{
+
+    [Fact]
+    public void StartGameShouldReturnNotNull()
+    {
+        IConfiguration config = new ConfigurationBuilder().Build();
+        var controller = new AimTrainerGameController(config);
+
+        var result = controller.StartGame(new GameStartRequest(Difficulty.EASY, new Vector2(1920, 1080)));
+        Assert.True(result != null);
+    }
+
+        // It currently fails, because it can't get config values (at least that's my guess)
+    // [Fact]
+    // public void StartGameEasyShouldReturn10Dots()
+    // {
+        // IConfiguration config = new ConfigurationBuilder().Build();
+        // var controller = new AimTrainerGameController(config);
+
+        // var result = controller.StartGame(new GameStartRequest(Difficulty.EASY, new Vector2(1920, 1080)));
+        // Assert.True(((result.Result as OkObjectResult).Value as GameStartResponse).dotInfos.Count == 10);
+    // }
```

Comment:
maine neturetu buti uzkomentuoto kodo pagal mane, nes jis gali uzsilikti pamirstas in the future, tai idk ar paliekam ji ar ne, your call